lib/MetaCPAN/Server/Controller/Login/Twitter.pm

```diff
@@ -24,39 +24,46 @@ sub index : Path {
     my ( $self, $c ) = @_;
     my $req = $c->req;
     if ( my $code = $req->parameters->{oauth_verifier} ) {
-        my $nt = $self->nt;
-        $nt->request_token( $c->req->cookies->{twitter_token}->value );
-        $nt->request_token_secret(
-            $c->req->cookies->{twitter_token_secret}->value );
+        my $nt       = $self->nt;
+        my $response = $nt->oauth_access_token(
+            token        => $c->req->cookies->{twitter_token}->value,
+            token_secret => $c->req->cookies->{twitter_token_secret}->value,
+            verifier     => $code,
+        );
 
-        my ( $access_token, $access_token_secret, $user_id, $screen_name )
-            = $nt->request_access_token( verifier => $code );
         $c->controller('OAuth2')->redirect( $c, error => 'token' )
-            unless ($access_token);
+            unless ( $response->{oauth_token_secret} );
+
         $self->update_user(
             $c,
-            twitter => $user_id,
+            twitter => $response->{user_id},
             {
-                id                  => $user_id,
-                name                => $screen_name,
-                access_token        => $access_token,
-                access_token_secret => $access_token_secret
+                id                  => $response->{user_id},
+                name                => $response->{screen_name},
+                access_token        => $response->{oauth_token},
```

Comment:
I think, for security, we no longer need to store `oauth_token` and `oauth_token_secret`. I don't believe we actually use the tokens anywhere else in the API.  @mickeyn?

Reply:
I don't see that we do anywhere in the code.